lectures/10.HPC_and_parallel_compute/SLURM_cheatsheet.md

```diff
@@ -0,0 +1,79 @@
+# Slurm Guide
+
+    
+For bash scripts this line should be the first line of code in every script 
+```
+#!/bin/bash  # Shebang slash bin slash bash
+```
+Next are the SBATCH commands that tell slurm scheduler how to handle your job
+
+### Frequent SBATCH Commands
+```
+#SBATCH --job-name=parallel_job # job name
+#SBATCH -t 1-23:59:59   # D-HH-MM-SS
+#SBATCH -t 59           # MM
+#SBATCH -t 59:59        # MM:SS
+#SBATCH -t 59:59:59     # HH:MM:SS
+#SBATCH -t 1-23         # D-HH
+#SBATCH -t 1-23:59      # D-HH-MM
+#SBATCH --mem=16G       # 16 Gigabytes
+#SBATCH --output=out_%j.log
+#SBATCH --ntasks     # number of tasks
+#SBATCH --mail-type=NONE, BEGIN, END, FAIL, ALL   # Mail events 
+#SBATCH --mail-user=email@ufl.edu
+```
+### Environment Modules
+```
```

Comment:
```suggestion
```

### Environment Modules

```
```

Convention is spaces around headers

Reply:
oh, looks like my suggestion was malformed because surrounded by backticks! (i hope my suggestion is clear)

Reply:
please do for remainder of this md file